### PR TITLE
feat(client): put markdown content and sponsors on home

### PR DESCRIPTION
**Pull request by ginkoid**

### CODE REVIEW

File: client/src/routes/home.js [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
   logo: {
     height: '9.375em !important',
     width: '9.375em !important'
+  },
+  content: {
+    '& a': {
+      display: 'inline',
+      padding: '0'
+    },
+    '& h3': {
```
Comment (chen-robert) [inline client/src/routes/home.js:21]: Are we only applying to h3 - as opposed to say h* - for a reason?

File: config/client.js.example [hunk 2]
```diff
@@ -8,16 +8,16 @@ module.exports = {
     description: 'A description of your CTF',
     imageUrl: ''
   },
-  logoUrl: 'https://webite.com/logo.png',
+  homeContent: 'A description of your CTF. Markdown supported.'
```
Comment (ethanwu10) [inline config/client.js.example:11]:
```suggestion
  homeContent: 'A description of your CTF. Markdown supported.',
```

File: client/src/routes/home.js [hunk 3]
```diff
@@ -1,33 +1,46 @@
...
 import withStyles from '../components/jss'
 
 export default withStyles({
-  root: {
-    padding: '3.375em'
-  },
   row: {
     marginBottom: '3.125em'
   },
   logo: {
```
Comment (ethanwu10) [inline client/src/routes/home.js:12]: Unused style

File: client/src/routes/home.js [hunk 4]
```diff
@@ -1,33 +1,46 @@
...
 import 'linkstate/polyfill'
+import Sponsors from '../components/sponsors'
+import Markdown from '../components/markdown'
 import withStyles from '../components/jss'
 
 export default withStyles({
-  root: {
-    padding: '3.375em'
-  },
   row: {
```
Comment (ethanwu10) [inline client/src/routes/home.js:9]: Unused style

File: client/lib/loaders/mustache-config-loader.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const mustache = require('mustache')
+
+module.exports = function (source) {
+  const config = require('../../../config/client')
+  return mustache.render(source, {
```
Comment (ethanwu10) [inline client/lib/loaders/mustache-config-loader.js:5]:
```suggestion
  // FIXME: refactor this (copy-pasted from server)
  return mustache.render(source, {
```

**Pull request merged**